opencv/Dockerfile

```diff
@@ -1,69 +1,59 @@
-FROM hudl/ffmpeg
-
+FROM hudl/ffmpeg:latest
 MAINTAINER Ryan Versaw <ryan.versaw@hudl.com>
 
-# Dependencies
 RUN apt-get update && apt-get install -y \
     cmake \
     gfortran \
     libatlas-base-dev \
-    python \
-    python-dev \
     wget
 
-# Download OpenCV and extra modules
+RUN apt-get install software-properties-common -y && \
+    add-apt-repository ppa:george-edison55/cmake-3.x && \
+    apt-get update && \
+    apt-get upgrade cmake -y
+
+RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O install_miniconda.sh && \
+    /bin/bash install_miniconda.sh -b -p /opt/conda && \
+    rm install_miniconda.sh
+ENV PATH /opt/conda/bin:$PATH
+
+RUN pip install --upgrade pip
+RUN pip install Cython==0.24
+RUN pip install --upgrade --ignore-installed https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0-cp35-cp35m-linux_x86_64.whl
```

Comment:
I don't want to make things overly complicated, but does it make sense to include `tensorflow` in an `opencv` image? Should that be a separate image that potentially builds off this one? The main reason I'm even bringing this up is that we consider this one of our public images, so it might be good to stay a bit more strict about what we include in them and how we name them.